apps/participants/utils.py

```diff
@@ -27,6 +27,11 @@ def get_participant_teams_for_user(user):
     return Participant.objects.filter(user=user).values_list("team", flat=True)
 
 
+def is_user_creator_of_participant_team(user, participant_team):
+    """Returns boolean if user is creator of Participant Team ids"""
```

Comment:
`Returns boolean if user is creator of Participant Team ids` -> `Returns boolean if a user is the creator of participant team`